moto/glue/models.py

```diff
@@ -136,9 +137,13 @@ def get_table(self, database_name, table_name):
         except KeyError:
             raise TableNotFoundException(table_name)
 
-    def get_tables(self, database_name):
+    def get_tables(self, database_name, expression=""):
```

Comment:
```suggestion
    def get_tables(self, database_name, expression):
```
The `expression`-parameter is always provided by `responses.py` - we just have to ensure we can deal with a None-value